### ISSUE TITLE
The AUC Result

### ISSUE THREAD

**Issue by Coalin** [ChenglongChen/tensorflow-DeepFM#29]
May I know why the AUC score in the Kaggle dataset and my own dataset is so strange that never runs above 0.5 ?  
Thanks a lot~ 
```
from sklearn.metrics import roc_auc_score

[1] train-result=0.0130, valid-result=0.0551 [33.9 s]
[2] train-result=0.2280, valid-result=0.2518 [35.6 s]
[3] train-result=0.2539, valid-result=0.2593 [33.7 s]
[4] train-result=0.2590, valid-result=0.2629 [34.0 s]
[5] train-result=0.2559, valid-result=0.2561 [36.0 s]
[6] train-result=0.2588, valid-result=0.2594 [34.4 s]
[7] train-result=0.2579, valid-result=0.2587 [33.2 s]
[8] train-result=0.2640, valid-result=0.2626 [33.6 s]
[9] train-result=0.2671, valid-result=0.2638 [34.8 s]
[10] train-result=0.2619, valid-result=0.2601 [35.0 s]
[11] train-result=0.2667, valid-result=0.2609 [36.3 s]
[12] train-result=0.2664, valid-result=0.2627 [37.6 s]
[13] train-result=0.2687, valid-result=0.2650 [34.1 s]
[14] train-result=0.2724, valid-result=0.2692 [33.9 s]
[15] train-result=0.2689, valid-result=0.2663 [34.0 s]
[16] train-result=0.2710, valid-result=0.2671 [33.2 s]
[17] train-result=0.2677, valid-result=0.2655 [33.1 s]
[18] train-result=0.2694, valid-result=0.2674 [34.5 s]
```

**Comment by chenyanyin** [comment]
what's happended? I had the same problem

**Comment by chenyanyin** [comment]
> what's happended? I had the same problem

i see, metric is different.

**Comment by WoNiuHu** [comment]
> > what's happended? I had the same problem
> 
> i see, metric is different.

是啥原因？

**Comment by alpha008** [comment]
我是健翔，您的邮件我收到了。
